engines/wintermute/base/gfx/opengl/base_surface_opengl3d.h

```diff
@@ -50,9 +50,7 @@ class BaseSurfaceOpenGL3D : public BaseSurface {
 	bool restore() override;
 	bool create(const Common::String &filename, bool defaultCK, byte ckRed, byte ckGreen, byte ckBlue, int lifeTime = -1, bool keepLoaded = false) override;
 	bool create(int width, int height) override;
-	bool putSurface(const Graphics::Surface &surface, bool hasAlpha = false) override {
-		return STATUS_FAILED;
-	}
+	bool putSurface(const Graphics::Surface &surface, bool hasAlpha = false);
```

Comment:
I assume this should still have override?

Reply:
Added override back